tutorial/create-a-simple-squid.md

```diff
@@ -125,6 +125,8 @@ To finalize this step, it is necessary to run the `codegen` tool, to generate Ty
 npx sqd codegen
 ```
 
+TODO.
```

Comment:
Should we mentioned that result of the `npx sqd codegen` is updated model under `src/model/generated`?